### PR TITLE
Update pulumi-aws dependency

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
It pulls in the fix from here: https://github.com/pulumi/pulumi-aws/pull/280

**Comment by ellismg** [comment]
Given that we don't actually depend on any new features from `@pulumi/aws` why do we force the version here? Is it not more idiomatic to just let the arrow constraint do its job and pick up the newer version when it is released?  /cc @lukehoban, since I don't actually understand what the "node way" is here.

File: aws/aws_test.go [hunk 1]
```diff
@@ -87,9 +87,9 @@ func Test_Examples(t *testing.T) {
 				for true {
 					elapsed := time.Now().Sub(start)
 
-					// lambdas can ony run up to 5 minutes.  So if we go to 6, then there's no point
+					// lambdas can ony run up to 10 minutes.  So if we go to 1, then there's no point
```
Comment (lukehoban) [inline aws/aws_test.go]: They can only run up to 5 minutes - right?

**Comment by lukehoban** [comment]
>  Is it not more idiomatic to just let the arrow constraint do its job and pick up the newer version when it is released?

Yes - unless there is a strict dependency on something added in the underlying version we should let the `^0.14.0` dependency resolve to the latest.

**Comment by CyrusNajmabadi** [comment]
> Yes - unless there is a strict dependency on something added in the underlying version we should let the ^0.14.0 dependency resolve to the latest.

Ok.  So that sounds like the *right* thing to do in terms of what we're actually delivering.  However, in terms of actually trying to use that locally, it's unclear to me how to make that actually work.  If we specify ^0.14.0 then all that will be pulled in locally is ^0.14.4, not ^0.14.5...  This is because ^0.14.4 is the 'latest', and that seems to be what yarn is willing to upgrade hte package to.

Locally, it's unclear to me how to actually get these libraries to actually be pulled forward (across all the repos) to this real "dev latest" that has been built...

**Comment by CyrusNajmabadi** [comment]
Not the right way to do things.

**Pull request closed without merging**